### DISCUSSION TITLE
Forme & Sequel Validation Helpers

### DISCUSSION THREAD

**Discussion by teamschlaflos** [jeremyevans/forme#120] [General]
Having trouble to work with Forme & Sequel validation helpers, as Sequel will add an error for each validation error. The generated error looks awful in the user interface. Any ideas on how to make this look better?

```html
<div class="invalid-feedback" id="proxy_port_error_message">is not present, Required, Invalid port</div>
```

Table & Model

```ruby
create_table(:proxies, options: [ :strict ] ) do
  Integer :port, null: false
end

class Proxy < Sequel::Model
  def validate
    super
    validates_presence [:port], message: "Required"
    validates_includes (1..65535), :port, message: "Invalid port"
  end
end
```

**Comment by jeremyevans** [comment]
For the general case, would you prefer a separate `div` for each error?

For this specific case, the `is not present` isn't coming from your code, are you using Sequel's auto_validations? You could probably avoid the `is not present, Required` issues by setting the `:required` option on the Forme input, so that the browser will not allow the form submission until the field is filled in.
